src/js/client/components/RecipeForm.js

```diff
@@ -205,6 +197,7 @@ export default class RecipeForm extends React.Component {
           />
           <a
             href="#"
+            onMouseOver={() => this.handleValidator(this.state.newRecipe)}
```

Comment:
@clayyy123 I feel like this shouldn't be checking the fields are valid on hover but also on click

Reply:
I changed it to Mouse Down because when Onclick happens it jumps to the the add ingredients page before triggering the validator